x-pack/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/kibana/enable_metricbeat_instructions.js

```diff
@@ -0,0 +1,274 @@
+                    defaultMessage="the instructions here"
+                  />
+                </EuiLink>
+              )
+            }}
+          />
+        </p>
+      </EuiText>
+    )
+  };
+
+  const enableMetricbeatModuleStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.kibanaInstructions.enableMetricbeatModuleTitle', {
+      defaultMessage: 'Enable and configure the Kibana x-pack module in Metricbeat'
+    }),
+    children: (
+      <Fragment>
+        <EuiCodeBlock
+          isCopyable
+          language="bash"
+        >
+          metricbeat modules enable kibana-xpack
+        </EuiCodeBlock>
+        <EuiSpacer size="s"/>
+        <EuiText>
+          <p>
+            <FormattedMessage
+              id="xpack.monitoring.metricbeatMigration.kibanaInstructions.enableMetricbeatModuleDescription"
+              defaultMessage="By default the module will collect Kibana monitoring metrics from http://localhost:5601. If the local Kibana instance has a different address, you must specify it via the {hosts} setting in the {file} file."
+              values={{
+                hosts: (
+                  <Monospace>hosts</Monospace>
+                ),
+                file: (
+                  <Monospace>modules.d/kibana-xpack.yml</Monospace>
+                )
+              }}
+            />
+          </p>
+        </EuiText>
+        {securitySetup}
+      </Fragment>
+    )
+  };
+
+  const configureMetricbeatStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.kibanaInstructions.configureMetricbeatTitle', {
+      defaultMessage: 'Configure Metricbeat to send to the monitoring cluster'
+    }),
+    children: (
+      <Fragment>
+        <EuiText>
+          <FormattedMessage
+            id="xpack.monitoring.metricbeatMigration.kibanaInstructions.configureMetricbeatDescription"
+            defaultMessage="Make these changes in your {file}."
+            values={{
+              file: (
+                <Monospace>metricbeat.yml</Monospace>
+              )
+            }}
+          />
+        </EuiText>
+        <EuiSpacer size="s"/>
+        <EuiCodeBlock
+          isCopyable
+        >
+          {`
+output.elasticsearch:
+  hosts: ["${esMonitoringUrl}"] ## Monitoring cluster
+
+  # Optional protocol and basic auth credentials.
+  #protocol: "https"
+  #username: "elastic"
+  #password: "changeme"
+`}
+        </EuiCodeBlock>
+        {securitySetup}
+      </Fragment>
+
+    )
+  };
+
+  const startMetricbeatStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.kibanaInstructions.startMetricbeatTitle', {
+      defaultMessage: 'Start Metricbeat'
+    }),
+    children: (
+      <EuiText>
+        <p>
+          <FormattedMessage
+            id="xpack.monitoring.metricbeatMigration.kibanaInstructions.startMetricbeatDescription"
+            defaultMessage="Follow {link}."
```

Comment:
Make the whole sentence the link including "Follow "

Reply:
Done, screenshot in previous similar comment